superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts

```diff
@@ -81,7 +81,6 @@ export const antDModalNoPaddingStyles = css`
   .ant-modal-body {
     padding-left: 0;
     padding-right: 0;
-    margin-bottom: 110px;
```

Comment:
I think we still want this margin just for the sqlalchemy view.

Reply:
just checking in, we are keeping this change or should i go back to 110px?